platform-includes/enriching-events/add-attachment/python.mdx

```diff
@@ -1,28 +1,22 @@
-Attachments live on the `Scope` and will be sent with all events.
+Attachments live on the [`Scope`](/platforms/python/enriching-events/scopes/). The SDK sends all attachments on a given `Scope` with every non-transaction event captured within the `Scope`. Optionally, an attachment can be sent with all events, including transactions, when the attachment is added with the `add_to_transactions` option set to `True`.
 
 ```python
 # Add an attachment
-from sentry_sdk import configure_scope
+from sentry_sdk import Scope
 
-with configure_scope() as scope:
-    scope.add_attachment(bytes=b"Hello World!", filename="attachment.txt")
-    scope.add_attachment(path="/path/to/attachment/file.txt")
-```
-
-An attachment has the following fields:
-
-`path`
+# Scope.get_isolation_scope() or Scope.get_global_scope() can also be used.
+# Read about the different scopes on our Scope docs page.
+scope = Scope.get_current_scope()
```

Comment:
I'd actually make this `get_isolation_scope` since for most cases attachments need to be scoped (like tags etc) to the isolation scope

Reply:
further, maybe we should also just expose `add_attachment` that adds it to isolation scope on the top level api now